.circleci/config.yml

```diff
@@ -700,7 +684,12 @@ jobs:
       - attach_workspace:
           at: .
       - run:
-          name: commit agent/uiserver/bindata_assetfs.go if there are UI changes
+          name: move compiled ui files to agent/uiserver
+          command: |
+            rm -rf agent/uiserver/dist
+            mv ui/packages/consul-ui/dist agent/uiserver
```

Comment:
This added step is now responsible for moving the compiled ui files to the correct workspace path